README.md

```diff
@@ -0,0 +1,84 @@
+![Swift version](https://img.shields.io/static/v1.svg?label=Swift&message=4.2&color=orange)
+![Platform support](https://img.shields.io/static/v1.svg?label=iOS&message=%3E=%2011.2&color=brightgreen)
+
+# Mapbox Vision SDK
+
+# Table of contents
+
+- [Overview](#overview)
+- [Getting Started](#getting-started)
+- [Documentation](#documentation)
+- [Contributing](#contributing)
+- [Versioning](#versioning)
+- [License](#license)
+
+# Overview
+
+The current repository is a part of [Vision SDK](https://vision.mapbox.com).
+
+The Vision SDK provides developers with cutting-edge AI and AR tools to build better driving experiences. It’s smart enough to understand the road, yet lean enough to run on devices that billions of drivers use everyday.
+
+The key features are:
+- Navigation in augemented reality;
+- Scene segmentation;
+- Sign detection;
+- Safety alerts;
+- Object detection;
+- Lane detection.
+
+Vision SDK for iOS contains several modules which built on top of Vision Core library (the implementation of Vision Core is in private repository):
+  1. `Vision SDK` - an end-user SDK with a lot of helpers such as:
+      * Device camera session implementation;
+      * Interfaces and additional functions for an API data structures;
```

Comment:
Not quite follow what it means.

Reply:
Me too :)
It was copied from Vision Architecture document. I suggest to remove that line.